config/application-example.yml

```diff
@@ -5,36 +5,31 @@ zuul:
     default:
       path: /**
       url: http://localhost:12345/
-# If you want to disable the user sync filter
-#  AcsUserSyncFilter:
-#    pre:
-#      disable: true
-logging:
-  level:
-    edu: TRACE
+  CertificateAuthenticationFilter:
+    pre:
+      disable: False
 
 gws:
   keystoreLocation: /keystore/location
   keystorePassword: password
 
 spring:
+  pid:
+    fail-on-write-error: false
   cache:
-      cache-names: acs-users, authorized-uri, user-groups, cert-retriever
-      caffeine:
-          spec: maximumSize=1000, expireAfterAccess=600s
+    type: none
```

Comment:
This is useful so you can modify the table on the fly without having to restart the server.